app/views/auth/signup.php

```diff
@@ -1,11 +1,123 @@
-<?php
-if (!empty($error)) {
-    echo $error;
-}
-?>
-<form action="" method="post">
-    Username: <input name="username" required /><br>
-    Password: <input type="password" name="password" required /><br>
-    Password: <input type="password" name="conf_password" required /><br>
-    <input type="submit" name="signup" value="Register" />
-</form>
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="<?php echo base_url(); ?>static/css/bootstrap.min.css" rel="stylesheet" media="screen">
+    <link href="<?php echo base_url(); ?>static/css/sidebar.css" rel="stylesheet" media="screen">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.2/jquery.min.js"></script>
+    <script src="<?php echo base_url(); ?>static/js/bootstrap.js"></script>
+
+</head>
+<nav class="navbar navbar-inverse navbar-fixed-top" id="navbar" role="navigation" style="visibility: visible">
+    <div class="container-fluid">
+
+        <!-- Brand and toggle get grouped for better mobile display -->
+        <div class="navbar-header">
```

Comment:
I thought we don't have to show the modules on the login and signup page, do we have to?